projects/example-app/src/app/books/effects/collection.effects.ts

```diff
@@ -73,5 +69,10 @@ export class CollectionEffects {
     )
   );
 
-  constructor(private actions$: Actions, private db: Database) {}
+  constructor(
+    private actions$: Actions<
```

Comment:
**lint-staged** always rewrite this block :(

how preferably to do in this situation?

Reply:
ok thanks